typescript/servicecatalog/portfolio-with-ec2-product/.npmignore

```diff
@@ -0,0 +1,6 @@
+*.ts
+!*.d.ts
+
+# CDK asset staging directory
+.cdk.staging
+cdk.out
```

Comment:
Mind if we remove this file if it's not necessary?

Reply:
Removed this.